hub/powertoys/run.md

```diff
@@ -160,21 +160,21 @@ The PowerToys Run calculator plugin supports the following operations:
 | Division |  a / b | |
 | Modulo/Remainder | a % b | |
 | Exponentiation | a ^ b | |
-| Ceil | ceil( x.y ) |  Rounds a number up to the next larger integer. |
+| Ceiling | ceil( x.y ) |  Rounds a number up to the next larger integer. |
 | Floor | floor( x.y ) | Rounds a number down to the next smaller integer. |
+| Round | round( x.abcd ) | Rounds a fractional number to the nearest integer. |
 | Exponential function | exp( x ) | Returns e raised to the specified power. |
 | Maximum | max( x, y, z ) | |
-| Minimum | max( x, y, z ) | |
-| Absolute | abs( - x ) | Absolute value of a number |
+| Minimum | min( x, y, z ) | |
+| Absolute | abs( -x ) | Absolute value of a number |
 | Log10 | log( x ) | |
 | Log base e | ln( x ) | |
 | Square root | sqrt( x ) | |
-| Power | pow( x, y ) | Calculate a number raised to the power of some other number. |
-| Factorial | x ! | |
-| Sign | sign( - x ) | A number that indicates the sign of value:<br />- `-1` if number is less than zero.<br />- `0` if number is zero.<br />- `1` if number is greater than zero. |
-| Round | round( x.abcd ) | Example: `round(8.7867)` |
-| Random | rand() | |
-| Pi | +pi | Returns the number of Pi. |
+| Power | pow( x, y ) | Calculate a number (x) raised to the power of some other number (y) |
+| Factorial | x! | |
+| Sign | sign( -x ) | A number that indicates the sign of value:<br />- `-1` if number is less than zero.<br />- `0` if number is zero.<br />- `1` if number is greater than zero. |
+| Random | rand() | Returns a fractional number between 0 and 1. |
+| Pi | pi | Returns the number of Pi. |
```

Comment:
The `+` sign is necessary. This wasn't a typo.

Reply:
You're right when typing it "standalone", like `=+pi`, but it could be confusing when you try `=6-pi`. Maybe the necessity itself is a bug?

Reply:
Yes. Can be a bug or a requirement of mages library.